examples/src/main/scala/org/apache/spark/examples/sql/streaming/KafkaWriteTest.scala

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Not sure why this file is here?

Reply:
Sorry, I was testing with it. I will remove it.